<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memberinfo</class>
 <widget class="QWidget" name="memberinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>会员功能</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="centerButton">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>个人中心</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="courseButton">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>课程预约</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="billButton">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>缴费记录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>650</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
